3_javascript/javascript1/week2/homework2ndweek/student.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Student manager</title>
+  </head>
+  <body>
+    <script>
+      const class07Students = [];
+
+      function addStudentToClass(studentName) {
+        if (class07Students.includes(studentName)) {
+          return (
+            "Student " + "" + studentName + " is already in the class" + "."
```

Comment:
this is more easy to use
`Student ${studentName} is already in the class.`

Reply:
its what was written in the task description 'Student Benjamin is already in the class', looks like string to me. I haven't use $ yet.

Reply:
Sorry for the confusion, the value you are returning is 100% correct, I just tried to give you an easier way to do it without all the + and " "
check this:
https://www.shecodes.io/athena/8931-creating-a-string-with-variables-in-javascript